std/src/main/scala/cats/std/string.scala

```diff
@@ -0,0 +1,8 @@
+package cats
+package std
+
+trait StringInstances extends algebra.std.StringInstances {
+
+  implicit val stringInstance: Show[String] = Show.show(identity)
```

Comment:
Very minor, but should we call this `stringShow` or something, since we have multiple instances for String (the others inherited from algebra)?